doc/swss-schema.md

```diff
@@ -669,6 +702,21 @@ Status: ready
 
 ## State DB schema
 
+### PORT_TABLE
+Stores information for physical switch ports managed by the switch chip. Ports to the CPU (ie: management port) and logical ports (loopback) are not declared in the PORT_TABLE. See MGMT_PORT.
+
+    ;State for layer 2 ports
+    port_table_key      = PORT_TABLE|ifname    ; ifname must be unique across PORT,INTF,VLAN,LAG TABLES
+    oper_status         = BIT           ; physical status up (1) or down (0) of the link attached to this port
```

Comment:
why the format is bit instead of string 'up' or 'down' since the in rest places, all admin/oper status are using 'up'/'down'.